crates/libs/bindgen/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "windows-bindgen"
+version = "0.49.0"
```

Comment:
> I've repurposed the windows-bindgen crate rather than minting yet another crate.

This is still the same version as that from 4 months ago:

https://crates.io/crates/windows-bindgen/versions

Looks like a publish didn't make its way through?

Reply:
Getting that ready now: #2611